### DISCUSSION TITLE
How to add a varchar/text field without a fixed length?

### DISCUSSION THREAD

**Discussion by duwejeferson** [drawdb-io/drawdb#130] [Q&A]
Hi.
First of all, awesome tool, thanks a lot for it.

So, to my problem, I'm trying to add some varchar and text fields without a fixed max length, but when a leave the length field empty this is what happens:
```sql
CREATE TABLE "table_name" (
    "id" bigserial NOT NULL,
    "varchar_without_fixed_length" varchar(), -- empty parentheses
    "varchar_7_length" varchar(7),	
    "text_without_fixed_length" text(),  -- empty parentheses
);
```
Is it possible to create those fields without a fixed length and the empty parentheses?

**Comment by DidulaGamage-lk** [comment]
CREATE TABLE "table_name" (
    "id" bigserial NOT NULL,
    "varchar_without_fixed_length" varchar,      -- No parentheses
    "varchar_7_length" varchar(7),               -- With length
    "text_without_fixed_length" text             -- No parentheses
);



Solution 1 - Don't Use Parentheses in the Length Field

When creating fields in drawdb:

varchar field - Leave the "length" parameter completely empty or don't enter a value
text field: Don't add any length specification

The generator should output varchar and text without parentheses when no length is specified.
